src/component/NodeEditor/NodeEditor.tsx

```diff
@@ -764,11 +765,9 @@ export default class NodeEditor extends React.PureComponent<NodeEditorProps, Nod
     }
 
     public open(): void {
-        const config = getTypeConfig(this.determineConfigType(this.props));
-
         this.setState({
-            show: true,
-            config
+            ...this.getInitialState(),
```

Comment:
yes eric, yes...embrace the spread operator...

Reply:
special case!